scripts/drop-database.js

```diff
@@ -0,0 +1,39 @@
+const { Client } = require('pg');
```

Comment:
See above about copy pasting. Did you try and complete the challenge first before adding this? If you did get stuck, thats okay, please dont take my comments as disparagement. Please feel free to ask for help/team up with others to work through these things together to help you get through the bits youre having trouble with